lib/sensu/utilities.rb

```diff
@@ -118,5 +118,69 @@ def substitute_tokens(tokens, attributes)
       end
       [substituted, unmatched_tokens]
     end
+
+    # Determine if a period of time (window) is subdued. The
+    # provided condition must have a `:begin` and `:end` time, eg.
+    # "11:30:00 PM", or `false` will be returned.
+    #
+    # @param condition [Hash]
+    # @option condition [String] :begin time.
+    # @option condition [String] :end time.
+    # @return [TrueClass, FalseClass]
+    def subdue_time?(condition)
+      if condition.has_key?(:begin) && condition.has_key?(:end)
+        begin_time = Time.parse(condition[:begin])
+        end_time = Time.parse(condition[:end])
+        if end_time < begin_time
+          if Time.now < end_time
+            begin_time = Time.parse("12:00:00 AM")
+          else
+            end_time = Time.parse("11:59:59 PM")
+          end
+        end
+        Time.now >= begin_time && Time.now <= end_time
+      else
+        false
+      end
+    end
+
+    # Determine if subdue conditions for one or more days of the week
```

Comment:
I think we should explicitly document that value of `:all`, if given, overrides any day-of-week specific configuration.